spec/changelog.yml

```diff
@@ -4,6 +4,9 @@
 ##
 - version: 3.4.0-next
   changes:
+    - description: Add support for required conditional groups of variables.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/855
```

Comment:
I think we could add this in 3.3.2. In versions of the stack that don't support this feature it will be likely ignored.